.expo/packager-info.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is your ```.expo``` folder necessary for your code base? If not, please ensure it is removed, [ here's why](https://www.youtube.com/watch?v=iF9W8CeCfNM&t=0s&index=5&list=PLEBnLDjR5B912lpuj_8kkqx8zxiiXQB61)